app/services/TwoStepVerificationThrottle.scala

```diff
@@ -23,7 +23,7 @@ import play.api.Play.{current, _}
 trait TwoStepVerificationThrottle {
   def timeBasedLimit: TimeBasedLimit
 
-  def isRegistrationMandatory(ruleName : String, discriminator: String) = {
+  def isRegistrationMandatory(ruleName: String, discriminator: String) = {
     val userValue = Math.abs((discriminator.hashCode % 1000).toDouble) / 10
```

Comment:
I know you didn't write these lines but can you give names to the numbers `1000` and `10`. @anshulbajpai gave them names in a recent PR for rule-engine